### PR TITLE
chore(i18n,learn): processed translations

### PR DESCRIPTION
**Pull request by camperbot**
This PR was opened auto-magically by Crowdin.

### CODE REVIEW

File: curriculum/challenges/portuguese/02-javascript-algorithms-and-data-structures/basic-data-structures/add-key-value-pairs-to-javascript-objects.md [hunk 1]
```diff
@@ -102,9 +102,9 @@ const foods = {
   plums: 28
 };
 
-// Only change code below this line
+// Altere somente o código abaixo desta linha
 
-// Only change code above this line
+// Altere somente o código acima desta linha
```
Comment (ShaunSHamilton) [inline curriculum/challenges/portuguese/02-javascript-algorithms-and-data-structures/basic-data-structures/add-key-value-pairs-to-javascript-objects.md:107]: I am not sure how this happened.

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
Something broke in our Crowdin config. The seed content should not be translatable.

**Comment by moT01** [comment]
Naomi and I had a meeting about this yesterday, something broke with how the markdown is being processed or something: https://community.crowdin.com/t/markdown-files-not-automatically-generating-context/5499/3

**Comment by moT01** [comment]
> it looks like we have new English curriculum files that need to be translated.

Can you elaborate on that @camperbot

**Pull request closed without merging**